GAS/cc_aarch64.S

```diff
@@ -656,6 +656,10 @@ fputc:
     pop x1
     ret
 
+enum_error_open_curly: .asciz "ERROR in enum\nExpected {\n"
```

Comment:
These `\n`  newlines  actually don't work  properly. See stage0-posix-amd64 for a fix.

Reply:
This is in `GAS`, does that also not work?

Reply:
Oh I meant to post it for M1 not for GAS, sorry.